aesara/link/jax/dispatch.py

```diff
@@ -108,6 +108,13 @@ def jax_typify_RandomState(state, **kwargs):
     return state
 
 
+@jax_typify.register(Generator)
+def jax_typify_Generator(state, **kwargs):
+    state = state.__getstate__()
+    state["bit_generator"] = numpy_bit_gens[state["bit_generator"]]
```

Comment:
Regarding the Jax conversion. I was thinking maybe we could generate a `PRNG` over here (and do the same with `RandomState`) and store it in the `state` dictionary as `jax_key`.

And in the `RandomVariable` conversion down below we could update this `jax_key` instead of the Numpy's state key.

Reply:
As long as `jax_key` stays localized to the JAX compilation process (i.e. we don't have to alter the generic `RandomState`/`Generator` types to explicitly account for it), we should be fine.